README.md

```diff
@@ -1352,6 +1353,68 @@ ha_cluster_cluster_properties:
 
 You may take a look at [an example](#configuring-acls).
 
+#### `ha_cluster_alerts`
+
+structure, default: no alerts
+
+```yaml
+ha_cluster_alerts:
+  - id: alert1
+    path: /alert1/path
+    description: Alert1 description
+    instance_attrs:
+      - attrs:
+        - name: alert_attr1_name
+          value: alert_attr1_value
+    meta_attrs:
+      - attrs:
+        - name: alert_meta_attr1_name
+          value: alert_meta_attr1_value
+    recipients:
+      - value: recipient_value
+        id: recipient1
+        description: Recipient1 description
+        instance_attrs:
+          - attrs:
+            - name: recipient_attr1_name
+              value: recipient_attr1_value
+        meta_attrs:
+          - attrs:
+            - name: recipient_meta_attr1_name
+              value: recipient_meta_attr1_value
+```
+
+This variable defines Pacemaker alerts.
+
+The items of `alerts` are as follows:
+
+* `id` (mandatory) - ID of an alert.
+* `path` (mandatory) - Path to the alert agent executable.
+* `description` (optional) - Description of the alert.
+* `instance_attrs` (optional) - List of sets of the alert's instance
+  attributes. Currently, only one set is supported, so the first set is used and
+  the rest are ignored.
+* `meta_attrs` (optional) - List of sets of the alert's meta attributes.
+  Currently, only one set is supported, so the first set is used and the rest
+  are ignored.
+* `recipients` (optional) - List of alert's recipients.
+
+The items of `recipients` are as follows:
+
+* `value` (mandatory) - Value of a recipient.
+* `id` (optional) - ID of the recipient.
+* `description` (optional) - Description of the recipient.
+* `instance_attrs` (optional) - List of sets of the recipient's instance
+  attributes. Currently, only one set is supported, so the first set is used and
+  the rest are ignored.
+* `meta_attrs` (optional) - List of sets of the recipient's meta attributes.
+  Currently, only one set is supported, so the first set is used and the rest
+  are ignored.
+
+**Note:** The role configures the cluster to call external programs to handle alerts. It is your responsibility to distribute the programs to cluster nodes.
```

Comment:
The user can use `ha_cluster_extra_packages` to list extra packages to install to provide those programs to the nodes.

Reply:
The "external programs" are e.g. custom scripts that reacts to cluster events and that are provided by user (the scripts doesn't have to be packages).
Thank you for pointing out this potentially confusing expression. It could be improved to `... It is your responsibility to provide the programs and distribute them to cluster nodes.` Does it work for you?

Reply:
> The "external programs" are e.g. custom scripts that reacts to cluster events and that are provided by user (the scripts doesn't have to be packages).

> Thank you for pointing out this potentially confusing expression. It could be improved to `... It is your responsibility to provide the programs and distribute them to cluster nodes.` Does it work for you?

Yes, thanks.

Reply:
Updated.